internal/controllers/reconciliation/edgecase_test.go

```diff
@@ -215,3 +218,113 @@ func TestLargeNamespaceDeletion(t *testing.T) {
 		return errors.IsNotFound(err)
 	}, time.Minute*3, time.Second)
 }
+
+// TestPatchStrategyReplace proves that resources which define a patch strategy of "replace" will eventually converge.
+func TestPatchStrategyReplace(t *testing.T) {
```

Comment:
Add a comment to specify that PDBs in particular have this issue? I forgot for a second and was at a loss as to what was being tested.